jujugui/static/gui/src/app/components/status/test-status.js

```diff
@@ -270,726 +73,33 @@ describe('Status', function() {
   });
 
   it('renders with no entities', () => {
-    const wrapper = renderComponent({ db: emptyDB });
-    assert.equal(wrapper.find('BasicTable').length, 1);
-  });
-
-  it('can display status by priority', () => {
-    const wrapper = renderComponent({ db: makeDB() });
-    assert.deepEqual(
-      wrapper.find('BasicTable').at(3).prop('rows')[1].classes,
-      ['status-view__table-row--error']);
+    db.machines.filter.returns([]);
+    db.relations.filter.returns([]);
+    db.remoteServices.map.returns([]);
+    db.services.filter.returns([]);
+    db.units.filter.returns([]);
+    const wrapper = renderComponent();
+    expect(wrapper).toMatchSnapshot();
   });
 
   it('renders with entities', () => {
```

Comment:
This doesn't actually test that it renders with entities as there is no data in the default stubs.